src/perform_step/rosenbrock_perform_step.jl

```diff
@@ -80,7 +80,7 @@ end
 
     @.. broadcast=false linsolve_tmp=f₁ - tmp
 
-    linres = dolinsolve(integrator, linres.cache; b = _vec(linsolve_tmp))
+    linres = dolinsolve(integrator, linres.cache; b = _vec(linsolve_tmp))/dtγ
```

Comment:
this is causing an allocation?

Reply:
Isn't it easier to just update the coeficients?

Reply:
yeah will fix.

Reply:
> Isn't it easier to just update the coeficients?

Computationally yes, but only once after writing the corrected version without changing the coefficients.

Reply:
redundant divisions removed.